projects/1.18/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+{
+  "block.stoneworks.andesite_brick_slab": "安山岩砖台阶",
+  "block.stoneworks.andesite_brick_stairs": "安山岩砖楼梯",
+  "block.stoneworks.andesite_brick_wall": "安山岩砖墙",
+  "block.stoneworks.andesite_bricks": "安山岩砖",
+  "block.stoneworks.andesite_paver_slab": "安山岩地砖台阶",
+  "block.stoneworks.andesite_paver_stairs": "安山岩地砖楼梯",
+  "block.stoneworks.andesite_paver_wall": "安山岩地砖墙",
+  "block.stoneworks.andesite_pavers": "安山岩地砖",
+  "block.stoneworks.andesite_pillar": "安山岩柱",
+  "block.stoneworks.andesite_plate_slab": "切制安山岩台阶",
+  "block.stoneworks.andesite_plate_stairs": "切制安山岩楼梯",
+  "block.stoneworks.andesite_plate_wall": "切制安山岩墙",
+  "block.stoneworks.andesite_plates": "切制安山岩",
+  "block.stoneworks.andesite_shingle_slab": "瓦制安山岩台阶",
+  "block.stoneworks.andesite_shingle_stairs": "瓦制安山岩楼梯",
+  "block.stoneworks.andesite_shingle_wall": "瓦制安山岩墙",
+  "block.stoneworks.andesite_shingles": "瓦制安山岩",
+  "block.stoneworks.andesite_tile_slab": "竹制安山岩台阶",
+  "block.stoneworks.andesite_tile_stairs": "竹制安山岩楼梯",
+  "block.stoneworks.andesite_tile_wall": "竹制安山岩墙",
+  "block.stoneworks.andesite_tiles": "竹制安山岩",
+  "block.stoneworks.basalt_brick_slab": "玄武岩砖台阶",
+  "block.stoneworks.basalt_brick_stairs": "玄武岩砖楼梯",
+  "block.stoneworks.basalt_brick_wall": "玄武岩砖墙",
+  "block.stoneworks.basalt_bricks": "玄武岩砖",
+  "block.stoneworks.basalt_paver_slab": "玄武岩地砖台阶",
+  "block.stoneworks.basalt_paver_stairs": "玄武岩地砖楼梯",
+  "block.stoneworks.basalt_paver_wall": "玄武岩地砖墙",
+  "block.stoneworks.basalt_pavers": "玄武岩地砖",
+  "block.stoneworks.basalt_plate_slab": "切制玄武岩台阶",
+  "block.stoneworks.basalt_plate_stairs": "切制玄武岩楼梯",
+  "block.stoneworks.basalt_plate_wall": "切制玄武岩墙",
+  "block.stoneworks.basalt_plates": "切制玄武岩台阶",
+  "block.stoneworks.basalt_shingle_slab": "瓦制玄武岩台阶",
+  "block.stoneworks.basalt_shingle_stairs": "瓦制玄武岩楼梯",
+  "block.stoneworks.basalt_shingle_wall": "瓦制玄武岩墙",
+  "block.stoneworks.basalt_shingles": "瓦制玄武岩",
+  "block.stoneworks.basalt_tile_slab": "竹制玄武岩台阶",
+  "block.stoneworks.basalt_tile_stairs": "竹制玄武岩楼梯",
+  "block.stoneworks.basalt_tile_wall": "竹制玄武岩墙",
+  "block.stoneworks.basalt_tiles": "竹制玄武岩",
```

```suggestion
  "block.stoneworks.basalt_tile_slab": "玄武岩瓦台阶",
  "block.stoneworks.basalt_tile_stairs": "玄武岩瓦楼梯",
  "block.stoneworks.basalt_tile_wall": "玄武岩瓦墙",
  "block.stoneworks.basalt_tiles": "玄武岩瓦",
```
见原版`深板岩瓦`/`Deepslate Tiles`
全文修改